recycleIT-frontend/src/Components/Header/Header.tsx

```diff
@@ -44,123 +56,152 @@ const Header:React.FC = () => {
   const handleMobileMenuOpen = (event: React.MouseEvent<HTMLElement>) => {
     setMobileMoreAnchorEl(event.currentTarget);
   };
+
+  let activeNavLinkStyle = {
+    borderBottom: "1.5px solid white",
+  };
+  let defaultNavLinkStyle = {
+    borderBottom: "1.5px solid transparent",
+  };
+
+  const renderMenu = (
+    <Box className={styles.Header_navbar_wrapper}>
+      <Box  className={styles.navbar_comtainer}>
+          <NavLink 
+            to="/" 
+            className={styles.Link}
+            style={({ isActive }) =>
+              isActive ? activeNavLinkStyle : defaultNavLinkStyle
+            }
```

Comment:
this code seems to be repeated for every NavLink, is there a way where it can be moved outside of render method and reused for all NavLink components? e.g.
```
const getNavLinkStyle = ({isActive}) => isActive ? activeNavLinkStyle : defaultNavLinkStyle;
// some other code...
<NavLink
  style={getNavLinkStyle}
/>
```